src/main/java/terminator/Main.java

```diff
@@ -0,0 +1,110 @@
+package terminator;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+/**
+ * Main GUI wrapper element for the application.
+ */
+public class Main extends Application {
+
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+
+    private Terminator terminator = new Terminator();
+
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/user.jpg"));
+    private Image dukeImage = new Image(this.getClass().getResourceAsStream("/images/terminator.jpg"));
+
+    /**
+     * Starts the GUI for the chatbot.
+     *
+     * @param stage The JavaFX Stage.
+     */
+    @Override
```

Comment:
Good job grouping the code, which helps to provide clear segmentation of the different segments of code. 
**_Improvements_**: Might want to provide clear and concise explanations of what each section of code is doing. 

For example: 
``` java
    // Configure user input and send button
    userInput.setPrefWidth(325.0);
    sendButton.setPrefWidth(55.0);
```